src/command/list/human.rs

```diff
@@ -58,46 +108,215 @@ fn display_all(
-    unimplemented!()
+    if package_managers.is_empty() {
+        String::from(
+            "⚡️ No <npm|Yarn> versions installed.
+
+You can install a Yarn version by running `volta install yarn`.
+See `volta help install` for details and more options.",
+        )
+    } else {
+        let versions = WRAPPER.fill(
+            &package_managers
+                .iter()
+                .map(format_package_manager)
+                .collect::<Vec<String>>()
+                .join("\n"),
+        );
+        format!("⚡️ Yarn versions in your toolchain:\n\n{}", versions)
+    }
 }
 
 /// Format a set of `Toolchain::Package`s and their associated tools.
 fn display_packages(packages: &[Package]) -> String {
-    unimplemented!()
+    if packages.is_empty() {
+        String::from(
+            "⚡️ No tools or packages installed.
+
+You can safely install packages by running `volta install <package name>`.
+See `volta help install` for details and more options.",
+        )
+    } else {
+        format!(
+            "⚡️ Package versions in your toolchain:\n\n{}",
+            format_package_list(packages)
+        )
+    }
 }
 
-/// Format the output for a specific tool from a set of `Toolchain::Package`s.
+/// Format a single `Toolchain::Tool` with associated `Toolchain::Package`
+
 fn display_tool(tool: &str, host_packages: &[Package]) -> String {
-    unimplemented!()
+    if host_packages.is_empty() {
+        format!(
+            "⚡️ No tools or packages named `{}` installed.
+
+You can safely install packages by running `volta install <package name>`.
+See `volta help install` for details and more options.",
+            tool
+        )
+    } else {
+        let versions = WRAPPER.fill(
+            &host_packages
+                .iter()
+                .map(format_package)
+                .collect::<Vec<String>>()
+                .join("\n"),
+        );
+        format!("⚡️ Tool `{}` available from:\n\n{}", tool, versions)
+    }
+}
+
+/// Format a list of `Toolchain::Package`s without detail information
+fn format_tool_list(packages: &[Package]) -> String {
+    packages
+        .iter()
+        .map(format_tool)
+        .collect::<Vec<String>>()
+        .join("\n")
+}
+/// Format a single `Toolchain::Package` without detail information
+fn format_tool(package: &Package) -> String {
+    match package {
+        Package::Default { tools, .. } | Package::Project { tools, .. } => {
+            let tools = match tools.len() {
+                0 => String::from(""),
+                _ => tools.join(", "),
+            };
+            WRAPPER.fill(&format!("{}{}", tools, list_package_source(package)))
+        }
+        Package::Fetched(..) => String::new(),
+    }
+}
+
+/// format a list of `Toolchain::Node`s.
+fn format_runtime_list(runtimes: &[Node]) -> String {
+    WRAPPER.fill(
+        &runtimes
+            .iter()
+            .map(|runtime| format_runtime(&runtime))
```

Comment:
Another _extremely_ minor nit: Can this work without the parameter? That is, can this be written as: `.map(format_runtime)` in the same way that `list_packages` does?